.gitignore

```diff
@@ -0,0 +1,11 @@
+###
+# Lock files. Arrange alphabetically by language
+###
+
+# JavaScript
+yarn.lock
+package-lock.json
```

Comment:
@SleeplessByte Could you alphabetise them pls? :D

Reply:
⏫ ⏬